### PR TITLE
Cards component  

### PR DESCRIPTION
**Pull request by SatwikaKacham**

### CODE REVIEW

**Comment by Kevin-Solomon** [comment]
Just a suggestion , the website would look much better if you fix the height issues of the iframe.

**Comment by mohammed-sas** [comment]
hey satwika ,in card with text-overlay make the text's background transparent

File: cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    align-content: center;
+   border:1px solid var(--secoundary);
+    width:14.1rem;
+    gap:0.5rem;
+    border-radius: 0.4rem;
+    box-shadow: 1.5px 1.5px 3px var(--secoundary);
+}
+/* section:not(footer){
+    background-color:#f1f1f3;  
+} */
```
Comment (kart-c) [inline cards/cards.css]: you can remove this code if it not required

File: cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    margin:1rem 0;
+}
+.highlight{
+    color:var(--primary);
+}
+.carbon-code{
+    box-sizing: border-box;
+    width: 100%;
+    min-height: fit-content;
+    max-height:20rem;
```
Comment (kart-c) [inline cards/cards.css]: On desktop site your carbon code looks too small 
Untitled
maybe you can change the height here

File: cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+background-color: white;
+margin:0;
+padding:0.5rem;
+cursor:pointer;
+border-bottom:1px solid var(--secoundary-black);
+}
+.card-heading-overlay:hover{
+    display:none;
+   
+}
```
Comment (kart-c) [inline cards/cards.css]: you can try using the `prettier` VS code extension to auto format your code

**Review by kart-c — CHANGES_REQUESTED** [review]
Hello Satvika I have suggested some changes. Apart from that everything else looks good

File: cards/cards.css (hunk 3 above)
Comment (SatwikaKacham) [reply cards/cards.css]: thank you so much for valuable review

**Review by Kevin-Solomon — APPROVED** [review]

**Review by kart-c — APPROVED** [review]

**Pull request merged**